src/test/java/com/wikia/webdriver/common/core/helpers/ForgottenPasswordUserFactory.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import lombok.Builder;
+import lombok.Getter;
+
+public final class ForgottenPasswordUserFactory {
+
+  private ForgottenPasswordUserFactory() {
+    // no-op
+  }
+
+  @Builder
+  public static class ForgottenPasswordUser {
+    private User user;
+
+    @Getter
+    private String email;
+
+    @Getter
+    private String emailPassword;
+
+    public String getUsername() {
+      return user.getUserName();
+    }
+
+    public String getPassword() {
+      return user.getPassword();
+    }
+  }
+
+  private static Credentials credentials = Configuration.getCredentials();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)